tests/transaction/test_get_list_transactions.py

```diff
@@ -0,0 +1,368 @@
+"""
+Provide tests for command line interface's get list of transactions command.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_IP_ADDRESS_FOR_TESTING,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+
+def test_get_list_transactions_with_ids():
+    """
+    Case: get a list transactions by ids.
+    Expect: transactions are returned.
+    """
+    transaction_ids = '044c7db163cf21ab9eafc9b267693e2d732411056c7530e54282946ec47cc180' \
+                      '201e7be5612a671a7028474ad18e3738e676c17a86b7180fc1aad4c97e38b85b, ' \
+                      '6601e240044b00db4b7e5eda7800e88236341077879a4a9cf5a1b1f9fb2ece87' \
+                      '7bc9a43808d429e68f4d65ee8d7231e4e8711e705ad51be7888d1a7f25b57717'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'transaction',
+        'get-list',
+        '--ids',
+        transaction_ids,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert isinstance(json.loads(result.output), dict)
+
+
+def test_get_list_transactions_with_invalid_ids():
+    """
+    Case: get a list transactions by invalid ids.
+    Expect: the following identifier are not valid error message.
+    """
+    invalid_id = '044c7'
+    transaction_ids = '044c7db163cf21ab9eafc9b267693e2d732411056c7530e54282946ec47cc180' \
+                      '201e7be5612a671a7028474ad18e3738e676c17a86b7180fc1aad4c97e38b85b, ' \
+                      f'{invalid_id}'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'transaction',
+        'get-list',
+        '--ids',
+        transaction_ids,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': {
+            'ids': [
+                f'The following identifier `{invalid_id}` is invalid.',
+            ],
+        },
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+def test_get_list_transactions_with_start():
+    """
+    Case: get a list transactions by start.
```

Comment:
`start` > `transaction identifier to get a list transaction starting from`.

Reply:
I mean in the documentation you should write user and business-oriented text, but in the code, you write details. 

For user, you write — transaction identifier to get a list transaction starting from.
In the code you just use — start.

Reply:
Apply this to all testing cases.

Reply:
@dmytrostriletskyi, done.